File: app/src/main/java/ru/ls/donkitchen/activity/categorylist/CategoryList.java
package ru.ls.donkitchen.activity.categorylist;

import android.support.v4.app.Fragment;

import ru.ls.donkitchen.activity.base.BaseActivity;

/**
 * @author Lord (Kuleshov M.V.)
 * @since 12.10.15
 */
public class CategoryList extends BaseActivity {
	@Override
	protected Fragment loadFragment() {
		return CategoryListFragment.newInstance() ;
	}
}

File: app/src/main/kotlin/ru/ls/donkitchen/activity/categorylist/CategoryList.kt
package ru.ls.donkitchen.activity.categorylist

import android.os.Bundle
import ru.ls.donkitchen.activity.base.BaseActivity

/**
 * Список категорий
 *
 * @author Lord (Kuleshov M.V.)
 * @since 17.10.16
 */
class CategoryList: BaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    override fun loadFragment() = CategoryListFragment.newInstance()
}
